e2e/package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "license": "BSD",
   "scripts": {
-    "test": "cross-env WORKBENCH_ENV=dev jest --detectOpenHandles",
+    "test": "cross-env WORKBENCH_ENV=dev jest",
```

Comment:
unrelated to imports. It's a small change to run tests in parallel for testing on localhost.